src/analysis/normed_space/spectrum.lean

```diff
@@ -347,6 +350,34 @@ begin
     (mem_resolvent_set_iff.mp (H₀.symm ▸ set.mem_univ 0)))),
 end
 
+section gelfand_mazur_isomorphism
+
+variables [normed_division_ring A] [normed_algebra ℂ A]
+
+local notation `σ` := spectrum ℂ
+
+open spectrum set
+
+lemma algebra_map_eq_of_mem {a : A} {z : ℂ} (h : z ∈ σ a) : algebra_map ℂ A z = a :=
+by rwa [mem_iff, is_unit_iff_ne_zero, not_not, sub_eq_zero] at h
+
+/-- **Gelfand-Mazur theorem**: For a complex Banach division algebra, the natural `algebra_map ℂ A`
+is an algebra isomorphism whose inverse is given by selecting the (unique) element of
+`spectrum ℂ a`.-/
+@[simps]
+noncomputable def _root_.normed_division_ring.alg_equiv_complex_of_complete
+  [complete_space A] [topological_space.second_countable_topology A] : ℂ ≃ₐ[ℂ] A :=
+{ to_fun := algebra_map ℂ A,
```

Comment:
Another way to state this would be `function.bijective (algebra_map ℂ A)`, which you can always conclude this result from as `alg_equiv.of_bijective`.

Reply:
I'll stick with the `alg_equiv` because it also says what the inverse is, and it's closer in spirit to the actual Gelfand-Mazur statement.